brackend/repositories/TournamentRepository.py

```diff
@@ -21,7 +21,7 @@ def get_all_for_user(cls, user):
                 .where(UserTournament.user_id == user.id) \
                 .all()
 
-            return tournaments
+            return [tournament.to_json() for tournament in tournaments]
```

Comment:
I'm not so sure about calling `to_json` in repository files. In my opinion, everything we pass around in the application code should remain SQL objects, especially something like `TournamentRepository.get_all_for_user`. 

`to_json` should really only be called at the last moment, when we actually want to send back json (I can't really think of a use case for passing around json within the app, but I can think of use cases for passing around the sqlalchemy objects). 

Note, also, we could get all this to_json for free with adding a marshaller - like marshmallow https://flask-marshmallow.readthedocs.io/en/latest/ - you can see an example of how a SQL alchemy object is converted directly to a json by defining a schema in the docs there, or a in-the wild example here: https://github.com/HourToEmpowerOrg/h2e-web/blob/main/backend/h2e_api/main/endpoints/sessions/schemas.py

Reply:
ok that makes sense, I was addressing this https://github.com/probicheaux/brackend/pull/21#discussion_r800097755

let me see if I can force loading of the brackets